scania-digital-design-system/sdds
# fix(button): corrected the styling of seconday,ghost and danger button

The focus style was wrong for all of the buttons above. The border should be 2px according to design, however it was 1px, needed to do some calculation to retain the correct height of the buttons on focus state. The secondary button also had the wrong background, border-color and color for hover-state.


**Describe pull-request**  
The focus style was wrong for all of the buttons above. The border should be 2px according to design, however it was 1px, needed to do some calculation to retain the correct height of the buttons on focus state. The secondary button also had the wrong background, border-color and color for hover-state.

**Solving issue**  
_In case of GitHub issue add # plus the number of the issue (for example #123) OR if it is Azure then AB# and number of ticket_
Fixes: AB#1820

**How to test**  
_Add description how to test if possible_
1. Go to storybook page below.
2. Check in the button component.
3. Compare the hover state of the secondary button and the focus state of secondary, ghost and danger buttons to the design (https://www.figma.com/file/d8bTgEx7h694MSesi2CTLF/Tegel-UI-Library?node-id=1574%3A72148). Also test it with the different sizes of the buttons.

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
Secondary hover state:
![Screenshot 2022-10-05 at 10 51 13](https://user-images.githubusercontent.com/62651103/194020694-4805a3d1-10de-4baf-81b1-0740b4d1af16.png)


Secondary, ghost and danger focus state:
![Screenshot 2022-10-05 at 10 52 28](https://user-images.githubusercontent.com/62651103/194020994-0084081b-70e0-48f7-ac5d-dcb229124f2b.png)

![Screenshot 2022-10-05 at 10 52 10](https://user-images.githubusercontent.com/62651103/194020913-9beba65d-8536-4115-b943-a317401aab41.png)

![Screenshot 2022-10-05 at 10 52 49](https://user-images.githubusercontent.com/62651103/194021077-0ff4b188-89d6-4765-a368-312241892fb4.png)
